jaxopt/_src/linear_solve.py

```diff
@@ -116,6 +123,32 @@ def solve_inv(matvec: Callable,
     raise NotImplementedError
 
 
+def solve_qr(matvec: Callable, 
+             b: jnp.ndarray, 
+             ridge: Optional[float] = None) -> tuple[jnp.ndarray, jnp.ndarray]:
```

Comment:
tuple -> Tuple (should solve failing tests)

Reply:
done.